.github/workflows/dev-ci-cd.yml

```
@@ -0,0 +1,122 @@
+          dockerfile: Dockerfile
+          push: true
+          tags: ${{secrets.DOCKER_USERNAME}}/server:latest
+
+
+  CD:
+    needs: CI
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: 1. Checkout source code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)